tools/wt_verify/wt_verify.py

```diff
@@ -586,12 +593,17 @@ def main():
 
     parsed_data = None
 
-    if "dump_pages" in command:
-        parsed_data = parse_dump_pages()
-    elif "dump_blocks" in command:
-        parsed_data = parse_dump_blocks()
-    else:
-        assert False, f"Unexpected command '{command}'"
+    try:
```

Comment:
After re-reading the code, I think from the `ArgumentParser`, we should never have an issue on the value:
```
    parser.add_argument('-d', '--dump', required=True, choices=['dump_blocks','dump_pages'], help='Option to specify dump_pages configuration.')
```
What are your thoughts on this @korteland ?